pkg/sdk/testint/external_functions_integration_test.go

```diff
@@ -215,46 +214,42 @@ func TestInt_ExternalFunctions(t *testing.T) {
 		otherDb, otherDbCleanup := testClientHelper().Database.CreateDatabase(t)
 		t.Cleanup(otherDbCleanup)
 
-		e1, _ := createExternalFunction(t)
+		e1 := createExternalFunction(t)
 
-		es, err := client.ExternalFunctions.Show(ctx, sdk.NewShowExternalFunctionRequest().WithIn(&sdk.In{Schema: e1.ID().SchemaId()}))
+		es, err := client.ExternalFunctions.Show(ctx, sdk.NewShowExternalFunctionRequest().WithIn(sdk.In{Schema: e1.ID().SchemaId()}))
 		require.NoError(t, err)
 
 		require.Contains(t, es, *e1)
 
-		es, err = client.ExternalFunctions.Show(ctx, sdk.NewShowExternalFunctionRequest().WithIn(&sdk.In{Database: testClientHelper().Ids.DatabaseId()}))
+		es, err = client.ExternalFunctions.Show(ctx, sdk.NewShowExternalFunctionRequest().WithIn(sdk.In{Database: testClientHelper().Ids.DatabaseId()}))
 		require.NoError(t, err)
 
 		require.Contains(t, es, *e1)
 
-		es, err = client.ExternalFunctions.Show(ctx, sdk.NewShowExternalFunctionRequest().WithIn(&sdk.In{Database: otherDb.ID()}))
+		es, err = client.ExternalFunctions.Show(ctx, sdk.NewShowExternalFunctionRequest().WithIn(sdk.In{Database: otherDb.ID()}))
 		require.NoError(t, err)
 
 		require.Empty(t, es)
 	})
 
 	t.Run("show external function: no matches", func(t *testing.T) {
```

Comment:
Added additional cases
- for integration test, same name with different arguments and different name with same arguments
- for acceptance tests I only tested procedures as resources use ShowByID that should be caught in the integration tests. In procedures I checked the case for argument synonyms that I changed in the new identifier with arguments constructor. Other resources were not affected, because they're not supporting them atm.